src/Bitshares/Operation.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Operation.h"
+
+#include <stdexcept>
+
+using namespace TW::Bitshares;
+using json = nlohmann::json;
+
+TransferOperation::TransferOperation(uint64_t senderId, 
```

Comment:
can you format this line? looks not good

Reply:
Okay, we'll run clang-format on it.